src/main/java/hudson/remoting/Channel.java

```diff
@@ -1865,7 +1916,7 @@ public static void dumpDiagnosticsForAll(@Nonnull PrintWriter w) {
      * Notification that {@link Command#readFrom} has succeeded.
      * @param cmd the resulting command
      * @param blockSize the serialized size of the command
-     * @see CommandListener
```

Comment:
IntelliJ and everything else, myself included, were absolutely confused about this reference.